synapse/lib/snap.py

```diff
@@ -154,7 +154,7 @@ async def iterStormPodes(self, text, opts=None, user=None):
         dorepr = False
         dopath = False
 
-        self.core._logStormQuery(text, user)
+        self.core._logStormQuery(text, user, opts.get('mode'))
```

Comment:
Couldn't we just make this default to storm rather than enforcing that every query has a default mode value